src/toggle.jsx

```diff
@@ -79,6 +79,7 @@ let Toggle = React.createClass({
       },
       label: {
         color: this.props.disabled ? this.getTheme().labelDisabledColor : this.getTheme().labelColor,
+        width: 'calc(100% - 46px)',
```

Comment:
Can you make the `46px` string less magic. I think that we can use `styles.icon.width + `10` to start.